src/core/utils/components/VirtualScroll.tsx

```diff
@@ -74,6 +92,20 @@ export type VirtualScrollProps = {
    * @default 10
    */
   bufferSize?: number;
+  /**
+   * Index of the first element on initial render.
+   * @default 0
+   */
+  scrollToIndex?: number;
+  /**
+   * Tag of the parent element. In most cases default `div` should be good enough.
+   * @default 'div'
+   */
+  as?: React.ElementType;
+  /**
+   * Props object to set on parent element.
+   */
+  parentProps?: React.ComponentPropsWithRef<T>;
```

Comment:
It can be a bit misleading as there are 2 parents. But I am not sure is this really a problem 😀

Reply:
updated description a little